openedx/core/djangoapps/discussions/views.py

```diff
@@ -61,4 +122,117 @@ def post(self, request, course_key_string: str, **_kwargs) -> Response:
                 check_course_permissions(course, request.user, 'change_provider')
 
             serializer.save()
-        return Response(serializer.data)
+        return serializer.data
+
+
+class DiscussionsProvidersView(APIView):
+    """
+    Handle configuration-related view-logic
+    """
```

Comment:
Would be useful to change to something like "read-only API for listing provider details".